src/contracts/core/AllocationManager.sol

```diff
@@ -738,6 +732,11 @@ contract AllocationManager is
         return operatorSets;
     }
 
+    /// @inheritdoc IAllocationManager
+    function isMemberOfOperatorSet(address operator, OperatorSet memory operatorSet) public view returns (bool) {
```

Comment:
Thinking we may also want introspection on until when the operator is slashable?